meetings/2024-02-28.md

```diff
@@ -0,0 +1,141 @@
+* MdJ: TBL, how do you feel about ACLs and SAI approach?
+* TBL: None of my code uses SAI. SolidOS uses WAC ACLs. I could imagine changing but the SAI way to use an app where it has an endpoint. So, need to have something on the Web. Mixing the page with the app. The world of trusted apps you do a lot of stuff like MS Word, Excel on the desktop where you give access. That's what people/consumers are used to. Collaborating with county members and doctors. If you build an app which I want to constrain to small amounts of things, that's reasonable. The whole process for running authentication agents gets unfortunately complicated. For example with bots, I got bots which do not need SAI. Like TimBLBot. I can give access to it using WAC or ACP. SAI to me looks complicated in the SolidOS world. Hackers? need Solid apps.
+* MdJ: Need to keep developing both ACL and Data grant based. Maybe up to the WG to decide in the end, which authn/z it likes.
+* TBL: What I like about consent is tracking the reasons why someone is given access to something.
+* eP: With SAI, we pay attention to privacy. So, authz first before discovery. So, another app or peer in the network, they are not able to discover the kind of information. To address privacy requirements, we need to do authz first.
+* MdJ: For example, in FB, everything is linked to your profile. Now, if you look at requirements for identity systems, it says don't leak global identity. Maybe want to use an app to give an access to bookmarks, but don't want to tell it my WebID.
+* SC: Henry Story was also touching on this quite a bit, presenting only the kind of credential that's needed to access something. Selective disclosure. This had to do with readable ACLs, extending WAC so the Controller can make the ACL readable without making it writable.
+* SC: And about reusing existing specs (such as OAuth), Solid in itself is not the most adequate place to solve those hard authentication/authorization problems for the web. Try to reuse and extend minimally.
+* eP: In some contexts, you don't want to use certain identifiers. In others, need to establish global identity. Many people use same name; they want to establish this kind of name/identity. So be relatable. Both directions are valid for different situations. Applicable to different use cases.
+* eP: prior work https://kushaldas.in/posts/solid-project-webid-and-privacy.html
+* TBL: Agree. Need to make sure the Solid community understands. People designing stuff for nuclear warheads.. ??? or someone linked to their online profiles. CC would give a token number, where it would work only once. Could that with WebID. Go to restaurant, happy ot authn as me, but also come up with my phone and have it generate a emphemeral ID where I'd only give my address. For whistleblowers, for example, where they dump some information.
```

Comment:
```suggestion
* TBL: Agree. Need to make sure the Solid community understands. People designing stuff for nuclear warheads.. ??? or someone linked to their online profiles. CC would give a token number, where it would work only once. Could that with WebID. Go to restaurant, happy to authn as me, but also come up with my phone and have it generate a ephemeral ID where I'd only give my address. For whistleblowers, for example, where they dump some information.
```